spread-tests/main/core-is-preferred/task.yaml

```diff
@@ -0,0 +1,10 @@
+summary: The snap named 'core' is preferred to the snap 'ubuntu-core'
+prepare: |
+    snap install --devmode snapd-hacker-toolbelt
+    snap install core
```

Comment:
Wouldn't `snap install ubuntu-core ; snap install core ; snap install --devmode snapd-hacker-toolbelt` be better? At some point, `snap install --devmode snapd-hacker-toolbelt` will pull 'core' instead of 'ubuntu-core' and then this test wouldn't be testing anything any more.